README.md

```diff
@@ -10,18 +10,38 @@ Run `npm run build:stats` to let source-map-explorer calculate the bundle size.
 
 ## Results:
 
-### [MiniRx Store](https://mini-rx.io/) (3.0.1)
+### [ngx-bang](https://github.com/nartc/nartc-workspace/tree/main/packages/ngx-bang) (1.0.0)
```

Comment:
MInor: Probably for future contribution / contribution guidelines, it would be better to leave the README untouched.

Maybe the result should be put into a result.txt or something similar.
After the MR has been merged I will update the README on master with the new result.